### ISSUE TITLE
Which wp Filter Plugins work well with Ele Custom Skin for custom post types?

### ISSUE THREAD

**Issue by GABRIELREIS8** [dudaster/ele-custom-skin#905]

**Comment by micmiller-apptio** [comment]
We use WP Grid Builder which has been very good for us. FacetWP also works well. Grid Builder was a better fit with the rest of our site architecture but I would suggest evaluating both if you haven't already.

**Comment by GABRIELREIS8** [comment]
micmiller thanks a lot for your response. Just to be sure, you are using WP Grid Builder and the loops are made with Ele Custom Skin right?

**Comment by micmiller-apptio** [comment]
Yes, that's correct. Our filters are done with WP Grid Builder and the loops are made with Elementor Post widgets using Ele Custom Skins for the loop item templates.

**Comment by GABRIELREIS8** [comment]
Thank you for your help!
